ngraph/frontend/paddlepaddle/src/frontend.cpp

```diff
@@ -288,10 +354,92 @@ namespace ngraph
         std::shared_ptr<ngraph::Function> FrontEndPDPD::convert(InputModel::Ptr model) const
         {
             auto pdpd_model = std::dynamic_pointer_cast<InputModelPDPD>(model);
-            auto f = convert_model(pdpd_model);
+            std::map<std::string, pdpd::CreatorFunction> CREATORS_MAP = pdpd::get_supported_ops();
+            auto f =
+                convert_each_node(pdpd_model,
+                                  [&](const std::map<std::string, Output<Node>>& nodes_dict,
+                                      const std::shared_ptr<OpPlacePDPD>& op_place) {
+                                      return pdpd::make_ng_node(nodes_dict, op_place, CREATORS_MAP);
+                                  });
+            return f;
+        }
+
+        std::shared_ptr<ngraph::Function>
+            FrontEndPDPD::convert(std::shared_ptr<ngraph::Function> partiallyConverted) const
+        {
+            auto function = clone_function(*partiallyConverted.get());
+            for (const auto& node : function->get_ordered_ops())
+            {
+                if (is_type<PDPDFrameworkNode>(node))
+                {
+                    pdpd::normalize_framework_node(
+                        std::dynamic_pointer_cast<PDPDFrameworkNode>(node),
+                        pdpd::get_supported_ops());
+                }
+            }
+            for (auto result : function->get_results())
+            {
+                result->validate_and_infer_types();
+            }
+            return function;
+        }
+
+        std::shared_ptr<ngraph::Function>
+            FrontEndPDPD::convert_partially(InputModel::Ptr model) const
+        {
+            auto pdpd_model = std::dynamic_pointer_cast<InputModelPDPD>(model);
+            std::map<std::string, pdpd::CreatorFunction> CREATORS_MAP = pdpd::get_supported_ops();
+            auto f = convert_each_node(
+                pdpd_model,
+                [&](const std::map<std::string, Output<Node>>& nodes_dict,
+                    const std::shared_ptr<OpPlacePDPD>& op_place) {
+                    pdpd::NamedOutputs named_outputs;
+                    try
+                    {
+                        named_outputs = pdpd::make_ng_node(nodes_dict, op_place, CREATORS_MAP);
+                    }
+                    catch (const OpConversionFailure&)
+                    {
+                        named_outputs = pdpd::make_framework_node(nodes_dict, op_place);
```

Comment:
This is also not covered by tests, and due to this `convert_partially` doesn't look fully tested